pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go

```diff
@@ -161,8 +161,16 @@ type TokenResponse struct {
 // KeycloakRealmStatus defines the observed state of KeycloakRealm
 // +k8s:openapi-gen=true
 type KeycloakRealmStatus struct {
-	Phase    string `json:"phase"`
-	LoginURL string `json:"loginUrl"`
+	// Current phase of the operator.
+	Phase StatusPhase `json:"phase"`
+	// Human-readable message indicating details about current operator phase or error.
+	Message string `json:"message"`
+	// True if all resources are in a ready state and all work is done.
+	Ready bool `json:"ready"`
+	// A map of all the secondary resources types and names created for this CR. e.g "Deployment": [ "DeploymentName1", "DeploymentName2" ]
+	SecondaryResources map[string][]string `json:"secondaryResources,omitempty"`
+	// TODO
```

Comment:
Is there anything more to be done in this PR?

Reply:
Can be done in this PR but probably better to tackle separately. I asked @pb82 about this on Thursday. He said it should be the URL that takes you to the realms login page, however he is not 100% sure how to construct it and was looking for feedback from yourself and @abstractj